README.md

```diff
@@ -1,21 +1,76 @@
-This repository holds a minimal installer for conda on platforms that
-conda-forge supports but that aren't supported by Miniconda
+# miniforge
+[![Build Status](https://travis-ci.org/conda-forge/miniforge.svg?branch=master)](https://travis-ci.org/conda-forge/miniforge)
 
+This repository holds a minimal installer for conda on platforms that conda-forge supports but that aren't supported by Miniconda.
 
-This work is still very experimental
+**Important: This work is still very experimental.**
 
-Relevant conversations
-* https://github.com/conda-forge/conda-forge.github.io/issues/871#issue-496677528
-* https://github.com/conda-forge/conda-forge.github.io/pull/922
+Relevant conversations:
 
+- https://github.com/conda-forge/conda-forge.github.io/issues/871#issue-496677528
+- https://github.com/conda-forge/conda-forge.github.io/pull/922
 
-Features of this repository to implement:
-- [ ] Automatic build of constructor
-    - [ ] aarch64 -- https://github.com/conda-forge/miniforge/pull/1
-    - [ ] Other architectures???
-- [ ] Automatic upload of constructor results
-    - [ ] Test: https://github.com/conda-forge/miniforge/pull/1
-- [ ] Automatic testing of constructor on Ubuntu 18.04 docker image
-- [ ] Integration with conda-forge's developer documentation
-- [ ] Integration with conda-forge's official site
-- [ ] Upstream to Anaconda???
+## Download
+
+Miniforge installers are available here: https://github.com/conda-forge/miniforge/releases.
+
+## Supported architectures
+
+- `aarch64` (also called `arm64`)
+
+## Features
+
+- [X] Automatic build of constructor.
+- [X] Automatic upload of constructor results.
+- [X] Automatic testing of constructor.
+- [ ] Integration with conda-forge's developer documentation.
+- [ ] Integration with conda-forge's official site.
+- [ ] Upstream to Anaconda ?
+
+## Testing
+
+After construction on Travis, the installer is tested against a range of distribution that match the installer architecture (`$ARCH`). For example when architecture is `aarch64`, the constructed installer is tested against:
+
+- Centos 7
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Ubuntu 19.10
+
+## Usage
+
+Installers are built and uploaded via Travis but if you want to construct your own Miniforge installer, here is how:
+
+```bash
```

Comment:
I suggest you simply move everything out of `.travis.yml` and point users there. This documentation will go "stale" otherwise.

Reply:
move out of .travils.yml into something like `build_miniconda.sh`

Reply:
With a parameter.

Reply:
Something like that:

```bash
export ARCH=aarch64
export DOCKERIMAGE=condaforge/linux-anvil-aarch64
export QEMU_BINARY=qemu-aarch64-static
build_miniconda.sh $ARCH $DOCKERIMAGE $QEMU_BINARY
```

???

and we use it in both travis and the README?

Reply:
Yeah